README.rst

```diff
@@ -96,10 +100,11 @@ The ADB is built on top of `CentOS 7`_ and the following projects:
 * `openshift2nulecule`_: a tool that creates a Nulecule application from an existing OpenShift project
 * `Mesos Marathon`_: a production-grade container orchestration platform for Mesosphere's Datacenter Operating System (DCOS) and Apache Mesos
 
-The ADB supports `Atomic App`_, an implementation of the multi-container
+ADB supports `Atomic App`_, an implementation of the multi-container
```

Comment:
We should probably delete these two lines given the current situation with nuleule and atomicapp -- @LalatenduMohanty